La-Mula-App-main/src/main/kotlin/com/lamulaapp/controller/dto/OrdersDto.kt

```diff
@@ -0,0 +1,19 @@
+package com.lamulaapp.controller.dto
+
+import com.lamulaapp.domain.OrderStatus
+import com.lamulaapp.domain.Users
+import java.io.Serial
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class OrdersDto(

```

Comment:
Las clases deben nombrarse en singular.

```suggestion
data class OrderDto(

```